crates/polars-core/src/fmt.rs

```diff
@@ -33,6 +33,7 @@ pub enum FloatFmt {
     Full,
 }
 static FLOAT_FMT: AtomicU8 = AtomicU8::new(FloatFmt::Mixed as u8);
+static FLOAT_PRECISION: AtomicU8 = AtomicU8::new(u8::MAX);
```

Comment:
Why is this an `u8`? There are floating point values that require more than 255 digits, e.g. `2^-1000` is exactly representable by an `f64` and requires 1000 decimal digits to print its full value.

Reply:
Digits vs precision; nothing here limits the total number of digits.

Reply:
I suppose we could make it arbitrary, but usefully there are only 16 digits of available precision in an `f64`, hence `u8`. I would be surprised if many people were hoping to print 1000 digit values, but I guess if somebody wants to and they have the terminal space, why stop them ;)